v3/how-to-guides/07-parachains/a-solo-to-parachain/index.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Convert a Solo-chain to a Parachain
```

Comment:
New HTG, not finished!! A placeholder mostly for now, will complete in followup PR is this lands

Reply:
@sacha-l this should not be a blocker here to get this in place, it's not rendered in the site yet either, this needs to be updated in the node and will be _after_ this PR is in and a _new_ PR updates the content before it's rendered. 

Can we approve on this?

Reply:
Please make follow-up PR so we can have the guide linked in and properly formatted according to the style guide. In the future it may make more sense to decouple such changes if one batch is incomplete. 👍🏻